lib/cmd_line.rb

```diff
@@ -0,0 +1,61 @@
+require_relative 'view_grid'
+require_relative 'update_grid'
+require_relative 'disk_based_memory'
+require_relative 'human_plays'
+require_relative 'ai_player'
+require_relative 'minimax'
+
+puts 'Start game? (y/n)'
+starting_input = gets.chomp
+
+def end_of_game?(memory)
+  memory.result == :winner || memory.result == :draw
+end
+
+def convert_to_cli(grid)
```

Comment:
I don't see a test for this :thinking:

Reply:
I thought we weren't meant to test the UI?

Reply:
`we weren't meant to test the UI?` 

I don't think it's clear cut - what is the cost of testing the UI, what is the value we get from testing the UI.

Are there parts of the code that can be tested easily (i.e. without the UI).

A lot of the code in this file does not need to depend on `gets` and `puts`, which means it can be tested easily ;)